Gopkg.toml

```diff
@@ -465,42 +465,42 @@ required = [
 
 [[constraint]]
   name = "k8s.io/api"
-  revision = "kubernetes-1.13.0-rc.1"
+  revision = "kubernetes-1.13.0-rc.2"
```

Comment:
I'm surprised that there are no new code changes pulled into the `vendor/` directory as a result of this.